core/src/main/scala/org/genivi/sota/core/CampaignResource.scala

```diff
@@ -72,6 +67,28 @@ class CampaignResource(namespaceExtractor: Directive1[Namespace],
     db.run(Campaigns.fetchMeta(id).map(_.namespace))
   }
 
+  def buildCampaignStats(groupId: Uuid, updateId: Uuid): Future[CampaignStatistics] = {
+    for {
+      rows <- updateService.fetchUpdateSpecRows(updateId)
+      groupSize <- deviceRegistry.fetchDevicesInGroup(groupId)
```

Comment:
fetchDevicesInGroup the end-point extract a namespace from the token, but here no token is sent.